src/Symfony/Component/Routing/Loader/AnnotationFileLoader.php

```diff
@@ -107,22 +107,22 @@ protected function findClass($file)
             }
 
             if (T_CLASS === $token[0]) {
-                // Skip usage of ::class constant
-                $isClassConstant = false;
+                // Skip usage of ::class constant and anonymous classes
+                $isValidClass = true;
```

Comment:
Not sure about this variable name -> `$isValidClass`  because an anonymous class is a valid class.

Reply:
I was mostly trying to avoid using `isClassConstantOrAnonymousClass` and `isValidClass` is the first thing that came to mind. I'll change it to something more reasonable